index.html

```diff
@@ -3,43 +3,314 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>Week 1: HTML Tables</title><!-- Latest compiled and minified CSS -->
+  <title>National Homlessness</title><!-- Latest compiled and minified CSS -->
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
   <style>
     .header {
       text-align: center;
-    }
+            } 
+    h1      {
+      color:blue;
+      font-family: "Trebuchet MS";
```

Comment:
`font-family` allows us to provide some fallbacks in case users don't have our first choice installed. So this could instead be: `font-family: "Trebuchet MS", Helvetica, sans-serif;`